test/test_stream_classes_model.rb

```diff
@@ -4,7 +4,7 @@ class TestSourceNoCommonField < Test::Unit::TestCase
   include TestSourceBase
   extend VariableAccessor
   include VariableClassAccessor
-
+  
```

Comment:
Please don't add space

Reply:
Do you mean remove the space between the include and the first method definition?

From this 

```ruby
  include VariableClassAccessor
  
  def self.startup
```
To this 

```ruby
  include VariableClassAccessor
  def self.startup
```

Reply:
No you just added some space is this line. This is what the `diff` show